tests/gatsby-starter-default/gatsby-config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
       options: {
         baseUrl: 'http://localhost:8080/Plone/',
         token: process.env.TOKEN,
-        showLogs: true,
+        logLevel: 'DEBUG',
```

Comment:
@ajayns I wanted more control on logging and copied some practices from Python logging module.

Reply:
This looks like an interesting feature, I haven't explored much Python.